change/@microsoft-teams-js-58d459f0-1a10-49ee-898f-117799bb4bda.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "removing hidden tag open function",
```

```suggestion
  "comment": "Added documentation for `stageView.open`",
```

Changefiles should be written in past tense and should describe what effect the change would have that would matter to external developers (because these comments get added to the monthly patch notes)

---

Also, references to code (like stageView.open) should be surrounded by backticks (`) so that github formats them differently